cmd/gb/env.go

```diff
@@ -13,37 +11,14 @@ func init() {
 
 var EnvCmd = &cmd.Command{
 	Name:      "env",
-	UsageLine: `env`,
+	UsageLine: `env [var ...]`,
 	Short:     "print project environment variables",
 	Long: `
-Env prints project environment variables.
+Env prints project environment variables. If one or more variable names is 
+given as arguments, env prints the value of each named variable on its own line.
 `,
-	Run: env,
-}
-
-func env(ctx *gb.Context, args []string) error {
-	env := makeenv(ctx)
-	for _, e := range env {
-		fmt.Printf("%s=%q\n", e.name, e.val)
-	}
-	return nil
-}
-
-type envvar struct {
-	name, val string
-}
-
-func findenv(env []envvar, name string) string {
-	for _, e := range env {
-		if e.name == name {
-			return e.val
-		}
-	}
-	return ""
-}
-
-func makeenv(ctx *gb.Context) []envvar {
-	return []envvar{
-		{"GB_PROJECT_DIR", ctx.Projectdir()},
-	}
+	Run: info,
+	ParseArgs: func(ctx *gb.Context, cwd string, args []string) []string {
+		return args
```

Comment:
a comment here would be nice

// env treats arguments as environment variables names, don't do any processing.